examples/train/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mxnet/python:gpu
+
+RUN apt-get update && \
+    apt-get install -y git && \
+    git clone https://github.com/apache/incubator-mxnet.git
```

Comment:
Do we need to check out a particular release to make surer this always works?

Reply:
Our base image uses mxnet/python:gpu which should be the latest version?  code is from master. I am ok to pin to a stable version like 1.5.x or 1.6.x. I will make the change